float64 reward
float64[] rewards
---
bool converged